lib/galaxy/jobs/runners/cli.py

```diff
@@ -177,6 +180,9 @@ def __get_job_states(self):
         for job_destination_id, v in job_destinations.items():
             job_destination = v['job_destination']
             job_ids = v['job_ids']
+            if not job_ids:
```

Comment:
It does happen in practice, I'll have to do some searching for how this is happening.